src/commons/BrowserUtils.ts

```diff
@@ -673,6 +673,29 @@ export namespace BrowserUtils {
     }
   }
 
+  /**
+   * Get cookie
+   * You can query a specific cookie by providing the cookie name or
+   * retrieve all.
+   */
+  export function getCookies(names?: string[] | string): WebDriver.Cookie[] {
+    Reporter.debug('Get cookie:');
+    const cookie: WebDriver.Cookie[] = tryBlock(() => browser.getCookies(names), 'Failed to get cookie');
+    Reporter.debug(`${JSON.stringify(cookie)}`);
```

Comment:
``${}`` its redundant here, you can use without it

Reply:
Done